README.md

```diff
@@ -0,0 +1,31 @@
+# myHPI
+
+This tool is used to manage the student representative website at https://myhpi.de (not live yet). It is a CMS based on Wagtail/Django and adds several functionalities like polls.
+
+## Development setup
+
+To set up a development version on your local machine, you need to execute the following steps:
+1. Check out repository and cd to it
+2. Set up a virtualenv for the project with Python >=3.8 and activate it
+3. Install poetry (if not already installed): `curl -sSL https://raw.githubusercontent.com/python-poetry/poetry/master/install-poetry.py | python -`
+4. Install dependencies with `poetry install`
+5. Create env file by copying the `.env.example file to `.env`, e.g. `cp .env.example .env`
+6. Migrate the database with `python manage.py migrate`
+7. Compile translations with `python manage.py compilemessages`
+8. Create a local superuser with `python manage.py createsuperuser`
+9. Start the development server with `python manage.py runserver`
+10. Open your web browser, visit `http://localhost:8000` and log in with your user from step 9
```

Comment:
Maybe we should also write how to create the first page? Because at the start this was very confusing for me